raw-spans-grouper/raw-spans-grouper/src/main/java/org/hypertrace/core/rawspansgrouper/RawSpansGrouper.java

```diff
@@ -59,18 +61,22 @@ public StreamsBuilder buildTopology(
 
     // Retrieve the default value serde defined in config and use it
     Serde valueSerde = defaultValueSerde(properties);
-    StoreBuilder<KeyValueStore<TraceIdentity, ValueAndTimestamp<RawSpans>>> traceStoreBuilder =
```

Comment:
make sure you have a plan to migrate i.e. wait until the inflight traces are flushed from the old store and then deploy

Reply:
I am thinking of following steps:
1. I will first bring down the existing service and allow all the existing traces to be flushed out. 
2. After that the new version will be deployed. 
3. Since the name of the new state store is different, the internal kafka changelog topics will be different and they will be automatically created. 
4. Old changelog topics will be deleted. 

I will also change the name of `TraceEmitStore` state store since its value has changed now